api/server.go

```diff
@@ -128,7 +130,10 @@ func (s *Server) CreateVault(c echo.Context) error {
 		return fmt.Errorf("fail to generate hex chain code, err: %w", err)
 	}
 
+	r := mathRand.New(mathRand.NewSource(time.Now().UnixNano()))
```

Comment:
This is a misunderstanding , I am thing the client means (UI part) , Can you please update the PR to allow client side(front end UI) to set a local Party id?